Screens/QudUX_GameStatsScreen.cs

```diff
@@ -0,0 +1,216 @@
+					new Table.ColumnDefinition {Header="Killed by",Width=31},
+				}
+			);
+
+			foreach(var sb in ScoreList)
+			{
+				string game =   sb.CharacterName + " " + sb.DeathDate.ToString("yyyy-MM-dd")  + "  "   + sb.Score.ToString()  + " " + sb.Level.ToString() +" " ;
+				//Logger.Log(game);
+				scoreTable.Rows.Add(new List<string> {sb.CharacterName  , sb.DeathDate.ToString("yyyy-MM-dd")  , sb.Score.ToString() , sb.Level.ToString(), sb.KilledBy });
+	
+			}
+
+			//
+			// levels stats table initialization			
+			//
+			
+			Table levelsTable = new Table(
+				new List<Table.ColumnDefinition>
+				{
+					new Table.ColumnDefinition {Header="Level",Width=10},
+					new Table.ColumnDefinition {Header="# Games",Width=10},
+				}
+			);
+
+			foreach(var sb in StatsByLevel)
+			{
+				levelsTable.Rows.Add(new List<string> {sb.Level.ToString()  , sb.Nb.ToString() });
+			}
+
+
+			//
+			// death cause stats table initialization			
+			//
+			
+			Table deathCauseTable = new Table(
+				new List<Table.ColumnDefinition>
+				{
+					new Table.ColumnDefinition {Header="Cause",Width=40},
+					new Table.ColumnDefinition {Header="# Games",Width=10},
+				}
+			);
+
+			foreach(var sb in StatsByDeathCause)
+			{
+				deathCauseTable.Rows.Add(new List<string> {sb.DeathCause , sb.Nb.ToString() });
+			}
+
+			while (true)
+            {
+                DisplayScreenFrame();
+				Buffer.Goto(55,0);
+				Buffer.Write(((int) currentScreen) .ToString() + "/3");
+
+				Table currentTable = null;
+                if (currentScreen == StatsScreen.GamesList)
+                {
+                    scoreTable.Display(Buffer, 1, 1);
+					currentTable = scoreTable;
+                }
+                else if (currentScreen == StatsScreen.LevelStats)
+                {
+					levelsTable.Display(Buffer, 1, 1);
+					currentTable = levelsTable;
+
+                }
+                else if (currentScreen == StatsScreen.DeathStats)
+                {
+					deathCauseTable.Display(Buffer, 1, 1);
+					currentTable = deathCauseTable;
+                }
+
+				    Buffer.Goto(2, 23);
+                    Buffer.Write(scoreboard.EnhancedScores.Count.ToString() + " {{O|games}}");
+                /*
+				// debug table
+				Buffer.Goto(1,24);
+				Buffer.Write($"si:{scoreTable.SelectedIndex} of:{scoreTable.Offset} mvr:{scoreTable.MaxVisibleRows} cvr:{scoreTable.CurrentVisibleRows} ");
+				*/
+
+                Console.DrawBuffer(Buffer);
+
+                Keys keys = Keyboard.getvk(Options.MapDirectionsToKeypad);
+
+                if (keys == Keys.Escape || keys == Keys.NumPad5)
+                {
+                    GameManager.Instance.PopGameView();
+                    return ScreenReturn.Exit;
+                }
+
+				if (currentTable != null)
+				{
+					if (keys == Keys.NumPad3)
+					{
+						currentTable.ScrollPage(1);
+					}
+
+					if (keys == Keys.NumPad9)
+					{
+						currentTable.ScrollPage(-1);
+					}
```

Comment:
Suggestion to also support PageUp and PageDown keys
```suggestion
					if (keys == Keys.NumPad3 || keys == Keys.Next)
					{
						currentTable.ScrollPage(1);
					}

					if (keys == Keys.NumPad9 || keys == Keys.Prior)
					{
						currentTable.ScrollPage(-1);
					}
```